pom.xml

```diff
@@ -2349,6 +2349,18 @@
                 <version>${hadoop.version}</version>
                 <scope>provided</scope>
             </dependency>
+            <dependency>
+                <groupId>org.apache.hadoop</groupId>
+                <artifactId>hadoop-mapreduce-client-common</artifactId>
+                <version>${hadoop.version}</version>
+                <scope>test</scope>
```

Comment:
looking at the dependency tree, we currently pull this in transitively as scope 'provided' in a bunch of places - could we mark it provided here?

Reply:
Hmm.... sure...